packages/frontend-shared/src/components/Tooltip.vue

```diff
@@ -77,6 +77,13 @@ const actualPopperClass = computed(() => {
   }
 }
 
+.v-popper__popper {
+  // Our modal overlays use a backdrop z-index of 10, but the tooltip lib defaults to 10000 which causes
+  // tooltips to overlay modals & modal backdrops. Override to set z-index so that tooltips aren't
+  // forced atop items they shouldn't be
+  z-index: 9;
```

Comment:
We have tooltips inside some popovers, which causes the tooltip to not overlay properly. Should this be a prop that gets passed in? We can default to a low z-index and than have an `in-modal` prop to bump the z-index up. There might be a clever way to detect the context of the tooltip and adjust

![Screen Shot 2022-07-28 at 10 06 02 AM](https://user-images.githubusercontent.com/25158820/181575257-52776260-2663-400a-acf0-67cf7680dee7.png)

Reply:
Dang, missed that in Percy.

I started off trying to use `z-index: inherit` so the tooltip could infer an appropriate layer based on its parent, but that caused issues with other areas since the tooltips are inherently ad-hoc DOM elements attached to `body`. I'll poke at it some more to see if there's a clever way to address via CSS